treeherder/seta/runnable_jobs.py

```diff
@@ -8,7 +10,7 @@
 
 class RunnableJobsClient():
     def __init__(self, treeherder_host='https://treeherder.mozilla.org',
```

Comment:
I know this is unrelated to this change, but I've just noticed `treeherder_host` is unused.